lesson11/style11.css

```diff
@@ -0,0 +1,237 @@
+@font-face {
+    font-family: whittelove;
+    src: url(White\ Love\ OTF.otf);
+}
+@font-face {
+    font-family: lobster;
+    src: url(Lobster-Regular.ttf);
+}
+@font-face {
+    font-family: merriweather;
+    src: url(Merriweather-BoldItalic.ttf);
```

Comment:
I see you have just 3 fonts in css, why you so many fonts files add to the project?